components/core/tests/test-ir_encoding_methods.cpp

```diff
@@ -1015,6 +1049,7 @@ TEMPLATE_TEST_CASE(
     // TODO: Test deserializing the serialized bytes once a KV-pair IR deserializer is implemented.
```

Comment:
```suggestion
    // TODO: Test validating the deserialized bytes once we've implemented a KeyValuePairLogEvent to
    // JSON deserializer.
```

Is the change accurate? Can we move it to the end of this function so that it's more clear that the validation doesn't actually check content right now?

Reply:
yeah makes sense